### PR TITLE
Feature/clean up subcommands

### PR DESCRIPTION
**Pull request by alexanderKoltman**

### CODE REVIEW

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/GetController.java [hunk 1]
```diff
@@ -54,64 +46,4 @@ public Integer call() throws Exception {
...
-            Map<String, ApplicationBean> applications = getLogic.getApplications(applicationsOperations);
-            System.out.println(YamlMapper.dump(applications));
-
-            return 0;
-        }
-    }
-
-}
\ No newline at end of file
+}
```
Comment (yenle2501) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/GetController.java:49]: you should add a new line hier

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/DiffController.java [hunk 2]
```diff
@@ -30,206 +25,51 @@
...
-            } else {
-                System.out.println(output);
-            }
-
-            return 0;
-        }
+        return 0;
     }
 
 }
```
Comment (yenle2501) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/DiffController.java:75]: you should add a new line

**Comment by yenle2501** [comment]
looks good to me, some tiny points should be changed :)

**Comment by felix-oq** [comment]
You should also consider removing all methods (and their test cases) in the GetLogic class except for the getAll-method

**Comment by M4eqP** [comment]
Please consider @yenle2501's and @felix-oq's remark before merging.

**Comment by alexanderKoltman** [comment]
> You should also consider removing all methods (and their test cases) in the GetLogic class except for the getAll-method. Also the demo files should be adjusted such that they are address commands that are still existing. Some Exception classes might also be obsolete by now.


Since the methods are still used in the ApplyController, I can not remove the methods in the Get-Logic.
The unit tests are therefore still required for this.

The exceptions are used in other ways so that none of the existing ones can be discarded.

The adaptation of the ApplyController is not in this scope, which is why it should be introduced as part of the applyAll().
In addition, I think it is good to maintain such fine granularity at the interfaces, but we can discuss it in the next task.

**Pull request merged**